src/serviceWorker.js

```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
+// register() is not called by default.
+
```

Comment:
- - Do you need this file? If not, please remove it as mentioned on Pathwright that you should delete any unnecessary file, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/useCreateAppAndDeleteUnusedFiles.png)

Reply:
I did make this change